gsheets-workflow-api/notebooks/exploration-notebook.py

```diff
@@ -42,10 +51,10 @@
 from lib.config import read_config
 
 config = read_config()
-gs_combined_spreadsheet_id = config["GS_COMBINED_SPREADSHEET_ID"]
+# gs_combined_spreadsheet_id = config["GS_COMBINED_SPREADSHEET_ID"]
 
 # Uncomment to use DEV spreadsheet during development
-# gs_combined_spreadsheet_id = config["GS_DEV_COMBINED_SPREADSHEET_ID"]
+gs_combined_spreadsheet_id = config["GS_DEV_COMBINED_SPREADSHEET_ID"]
```

Comment:
Avoid committing this change

Reply:
ok, changed back: https://github.com/Gapminder/gapminder-igno-survey-processing/pull/43/files#diff-d71f13d15ac19f01a99c74d74c02f5bf77a899f99949d251b96c0dd1cc494d53R59